sdk/face/azure-ai-vision-face/azure/ai/vision/face/_patch.py

```diff
@@ -0,0 +1,1044 @@
+                                  Required.
+                            },
+                            "noseTip": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            },
+                            "pupilLeft": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            },
+                            "pupilRight": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            },
+                            "underLipBottom": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            },
+                            "underLipTop": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            },
+                            "upperLipBottom": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            },
+                            "upperLipTop": {
+                                "x": 0.0,  # The horizontal component, in pixels.
+                                  Required.
+                                "y": 0.0  # The vertical component, in pixels.
+                                  Required.
+                            }
+                        },
+                        "recognitionModel": "str"  # Optional. The 'recognitionModel'
+                          associated with this faceId. This is only returned when
+                          'returnRecognitionModel' is explicitly set as true. Known values are:
+                          "recognition_01", "recognition_02", "recognition_03", and "recognition_04".
+                    }
+                ]
+        """
+        # By default, return_face_id is False.
+        if return_face_id is not True:
+            return_face_id = False
```

Comment:
`return_face_id` is typed as `bool` which is required keyword-only argument. So these 2 lines are surplus.

Reply:
Removed the if condition.